pkg/deployment/const.go

```diff
@@ -47,4 +47,10 @@ const (
 
 	// ConfigPaths base path for volume mounts in OpenStackAnsibleEE pod
 	ConfigPaths = "/var/lib/openstack/configs"
+
+	// RootCAIssuerInternalLabel for internal RootCA to issue internal TLS Certs
+	RootCAIssuerInternalLabel = "osp-rootca-issuer-internal"
```

Comment:
Do we need to support user specified CA cert or root? I vaguely remember discussing this with @stuggi before.

Reply:
Thats a good question, but I think that at least initially the answer to this is no.  The compute nodes do not have a public interface, so all the interfaces are internal.  All internal certs right now are being served from a well-defined osp-rootca-issuer

Reply:
@stuggi  ^^

Reply:
right, for internal we do not plan (at least right now) to support custom CA, we do for public. We could do the same for internal if we see there is the need. I'll have added this label in lib-common https://github.com/openstack-k8s-operators/lib-common/pull/356

Reply:
using the merged label now